public/app/core/hoc/withErrorBoundary.tsx

```diff
@@ -0,0 +1,17 @@
+import { ErrorBoundaryAlert } from '@grafana/ui';
+import React, { ComponentType } from 'react';
+import { WithAlertBoxProps } from '@grafana/ui/src/components/ErrorBoundary/ErrorBoundary';
```

Comment:
This should not be imported this way. You need to export if from `grafana/ui` and import it via the regular package import.

Reply:
done! Also renamed to `ErrorBoundaryProps` to be consistent with everything else